retail-managment-system/rms-ui/src/app/auth/components/login/login.component.scss

```diff
@@ -0,0 +1,40 @@
+mat-card {
+    max-width: 600px;
+    margin: 2em auto;
+    text-align: center;
+    max-height: 600px;
+  }
+  
+  .header{
+    text-align: center;
+    font-weight: bold;
+    font-size: large;
+  
+  }
+  .full-width {
+    width: 80%;
+  }
+  
+  .button-row {
+    padding-top: 5px;
+  }
+  
+  .button-row a {
+    margin-right: 8px;
+    text-align: center;
+  }
+  
+  .forget-password{
+    padding-left: 0px;
```

Comment:
there is this style in bootstrap why didn't used it?
and some others in this file.